CMakeLists.txt

```diff
@@ -12,8 +12,22 @@ add_definitions("-DLIBCIMBAR_PROJECT_ROOT=\"${libcimbar_SOURCE_DIR}\"")
 
 if("${CMAKE_BUILD_TYPE}" STREQUAL "")
 	set(CMAKE_BUILD_TYPE "RelWithDebInfo")
-	set(CMAKE_C_FLAGS "${CFLAGS} -Wall -std=c99 -Os")
-	set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-g -O2 -fpic")
+	set(CMAKE_C_FLAGS "${CFLAGS} -Wall -std=c99 -Os -fPIC")
+	set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-g -O2 -fPIC")
+endif()
+
+if(DEFINED USE_WASM)
+	set(DISABLE_TESTS true)
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGF256_TARGET_MOBILE")
+	set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-Os")
+
+	file( GLOB OPENCV_LIBS "${OPENCV_DIR}/opencv-build-wasm/build_wasm/lib/*.a" )
```

Comment:
This path is kind of nonsense (you have to build under `opencv-build-wasm`) -- need to clean this up before merging.